bigtable/quickstart.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START bigtable_quickstart]
+# Import google bigtable client lib
+require "google-cloud-bigtable"
+
+gcloud = Google::Cloud.new
+bigtable = gcloud.bigtable
+
+# Get table client
+table = bigtable.table("my-bigtable-instance", "my-table")
```

Comment:
Elsewhere in the repo, we avoid parenthesis around method calls when possible

Reply:
I agree. I think this should be addressed with an explicit Rubocop configuration, per #68.